docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx-autodoc-typehints
+sphinx-rtd-theme
+
```

Comment:
hint: as we already specify this in the `docs` extra in `setup.py`, I don't think we need this file (rtd allows to specify `extra_requirements`)